publish-token/src/main/java/fi/thl/covid19/publishtoken/Validation.java

```diff
@@ -1,6 +1,7 @@
 package fi.thl.covid19.publishtoken;
 
 import fi.thl.covid19.publishtoken.error.InputValidationException;
+import org.springframework.web.context.request.NativeWebRequest;
```

Comment:
I'd prefer to keep this class "clean" in the sense that it doesn't understand data sources, just data. Or in this case, not requests but just Strings.

Reply:
Ok, I see it more like a util collection of static methods which do different kinds of validation related things. Anyway, this method can be moved to somehere else, there is no much sense for it if it just have data anyway, but let's see how this will be developing.